packages/charts/CHANGELOG.md

```
@@ -0,0 +1,7 @@
+Changelog
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)